### PR TITLE
DOCKER: Add configuration to build production image

### PR DESCRIPTION
**Pull request by ambikads**
The dev Dockerfile is renamed to `Dockerfile.dev`. 
Non dev builds will be built from `Dockerfile`

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,12 @@ FROM node:13
...
+
+COPY package.json yarn.lock ./
 RUN yarn install
 
-CMD yarn dev -p 3000
+COPY . .
+RUN npm run build
+
+EXPOSE 3000
+CMD npm start -p 3000
```
Comment (markrawls) [inline frontend/Dockerfile:13]: Switching between `yarn` and `npm` in the middle of the Dockerfile is a bit suboptimal

File: frontend/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,12 @@ FROM node:13
...
 RUN mkdir /code
 WORKDIR /code
-ADD package.json /code/
+
+COPY package.json yarn.lock ./
 RUN yarn install
 
-CMD yarn dev -p 3000
+COPY . .
+RUN npm run build
```
Comment (markrawls) [inline frontend/Dockerfile:10]: This belongs in `CMD` - `CMD yarn build && yarn start -p 3000`

File: frontend/Dockerfile (hunk 1 above)
Comment (markrawls) [inline frontend/Dockerfile:13]: If I remember correctly, EB will choose a port dynamically at runtime and put it in `$PORT`?

**Comment by markrawls** [comment]
This probably belongs in a `willing-zg` plugin

File: frontend/Dockerfile (hunk 2 above)
Comment (ambikads) [reply frontend/Dockerfile]: `yarn start` includes `build`

File: frontend/Dockerfile (hunk 1 above)
Comment (ambikads) [reply frontend/Dockerfile]: We need to expose a port for docker platform

Comment (ambikads) [reply frontend/Dockerfile]: done

**Comment by ambikads** [comment]
@markrawls addressed comments

**Pull request merged**